src/components/custom/eui/EUIIntegration.js

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useRef } from "react";
+import Script from "next/script";
+
+const EUIIntegration = ({ height }) => {
+    const euiRef = useRef(null);
+
+    useEffect(() => {
+        const eui = euiRef.current;
+        eui.dataSources = [
+            "https://apps.humanatlas.io/hra-api/v1/sennet/rui_locations.jsonld",
+        ];
+    }, []);
+
+    return (
+        <>
+            <ccf-eui
+                style={{ height: height }}
+                ref={euiRef}
+                theme="sennet"
+                header="false"
+                use-remote-api="false"
```

Comment:
To get best performance, I have created a remote api for sennet to use. You can change `use-remote-api` to true, remove the data sources lines (7-12) above, and put the following as attributes on the ccf-eui web component:

```
use-remote-api="true"
remote-api-endpoint="https://apps.humanatlas.io/sennet-hra-api/v1"
```